airflow/providers/odbc/hooks/odbc.py

```diff
@@ -99,15 +92,6 @@ def sqlalchemy_scheme(self) -> str:
             raise RuntimeError("sqlalchemy_scheme in connection extra should not contain : or / characters")
         return self._sqlalchemy_scheme or extra_scheme or self.DEFAULT_SQLALCHEMY_SCHEME
 
-    @property
```

Comment:
Those removals will require to add common.sql  >= next.version in dependencies of providers that have them removed.

Reply:
Done